### PR TITLE
[BAL-2025] setup baltimore 2025 site

### PR DESCRIPTION
**Pull request by dsueltenfuss**
Creating the site for the 2025 Baltimore even

### CODE REVIEW

File: static/events/2025-baltimore/bmore_skyline_inner_harbor.jpg
Comment (toshywoshy) [inline static/events/2025-baltimore/bmore_skyline_inner_harbor.jpg:1]: Please move to the new folder `assets/events/2025-baltimore/`

File: static/events/2025-baltimore/logo-bmore.png
Comment (toshywoshy) [inline static/events/2025-baltimore/logo-bmore.png:1]: Please move to the new folder `assets/events/2025-baltimore/`

File: static/events/2025-baltimore/organizers/charles-bushong.jpg
Comment (toshywoshy) [inline static/events/2025-baltimore/organizers/charles-bushong.jpg:1]: Please move to the new folder `assets/events/2025-baltimore/`

File: static/events/2025-baltimore/organizers/dave-sueltenfuss.jpg
Comment (toshywoshy) [inline static/events/2025-baltimore/organizers/dave-sueltenfuss.jpg:1]: Please move to the new folder `assets/events/2025-baltimore/`

File: static/events/2025-baltimore/organizers/jp-hechter.png
Comment (toshywoshy) [inline static/events/2025-baltimore/organizers/jp-hechter.png:1]: Please move to the new folder `assets/events/2025-baltimore/`

File: static/events/2025-baltimore/organizers/judy-johnson.jpg
Comment (toshywoshy) [inline static/events/2025-baltimore/organizers/judy-johnson.jpg:1]: Please move to the new folder `assets/events/2025-baltimore/`

File: static/events/2025-baltimore/pava-center.jpg
Comment (toshywoshy) [inline static/events/2025-baltimore/pava-center.jpg:1]: Please move to the new folder `assets/events/2025-baltimore/`

**Review by toshywoshy — CHANGES_REQUESTED** [review]
- Please move to the new folder `assets/events/2025-baltimore/`
- Please email info@

**Comment by phrawzty** [comment]
organiser email received and processed

**Review by phrawzty — APPROVED** [review]

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**